cmd/merge-nodes/internal/merge_action_test.go

```diff
@@ -267,6 +271,10 @@ func Test_MergeDBs_Successful_Existing_Node(t *testing.T) {
 	err = os.WriteFile(filepath.Join(tmpSrc, keyDir, "id2.key"), key, 0o600)
 	require.NoError(t, err)
 
+	// this file should be ignored
+	err = os.WriteFile(filepath.Join(tmpSrc, keyDir, ".DS_Store"), types.RandomBytes(20), 0o600)
+	require.NoError(t, err)
```

Comment:
Shouldn't the test check if this file wasn't copied to the destination directory?

Reply:
Yes and no; the issue that the user brought up was that in both folders there was a `.DS_Store` file which prevented the tool from merging the nodes because it won't copy files that already exist at the destination.

I can add an additional assertion that the contents are unchanged.